File: LoopConstruction/C++/ArithmeticProgression.cpp
#include <iostream>
using namespace std;

int main() {
    int first_term, common_diff, n, term, sum = 0;

    cout << "Enter the first term of the A.P.: ";
    cin >> first_term;

    cout << "Enter the common difference: ";
    cin >> common_diff;

    cout << "Input number of terms in the series: ";
    cin >> n;

    cout << "Arithmetic Progression : ";
    for (int i = 0; i < n; i++) {
        term = first_term + i * common_diff;
        cout << term << " ";
        sum += term;
    }

    cout << "\nThe sum of the AP series is: " << sum << endl;
    return 0;
}

File: LoopConstruction/Java/ArithmeticProgression.java

import java.util.Scanner;

public class ArithmeticProgression {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("ENter first term: ");
        int firstterm = sc.nextInt();

        System.out.println("ENter Common differnce: ");
        int commondiff = sc.nextInt();

        System.out.println("ENter nth term: ");
        int n = sc.nextInt();
        int sum = 0;

        for(int i = 1; i<=n; i++){
            System.out.print(firstterm+" ");
            sum = firstterm + sum;
            firstterm += commondiff;
        }
        System.out.println();
        System.out.println("Total Sum : "+sum);
    }
}
